README.md

```diff
@@ -76,8 +78,11 @@ git clone github.com/Muhammad0602/Portfolio-mobile-first.git
 
 ### Usage
 
-To run the project, execute the following command:
- open the project by launching the Live Server plugging in visual studio code(or your code editor) 
+To run the project, execute the following commands:
+ - first `npm install`
+ - second `npm run build`
+ - third `npm start`
+ - and lastly for test purposes `npm install --save-dev jest-environment-jsdom`
 
```

Comment:
- [ ] [OPTIONAL]
Kindly include appropriate instructions in your ReadMe on how to run test cases your app, under the _Getting Started_ section, for instance using the command:
`npm test` or `npx jest .`